packages/aura/src/color.css

```diff
@@ -96,6 +77,36 @@
   background-attachment: fixed;
 }
 
+:where(:root, :host),
+vaadin-button,
+vaadin-menu-bar-button,
+vaadin-drawer-toggle,
```

Comment:
Not a blocker for this PR, just a general observation: we have some lists of components both in `color.css` and also `surface.css`, there can be presumably more of these?

While overall the code is quite self-explanatory, it would not harm to write down some short clarifications behind these at least in comments (they are minified anyways), some sort of file overview etc.